### ISSUE TITLE
Bootstrap 4.2.1. breaks jQuery scrollTop()

### ISSUE THREAD

**Issue by foss-friendly** [twbs/bootstrap#27947]
Dear Bootstrap,

My one-page website is using jQuery to make a transparent navbar go solid when the user scrolls below the landing section, plus change the color of navbar text. The code below was working till Bootstrap 4.1.3 and the 4.2.x series broke it. I also haven't touched the HTML, nor my style.css file. Is this a bug or I am missing something important from the 4.2.x release notes?

	$( window ).scroll( function () {
		if ( $( this ).scrollTop() > 500 ) {
			$( '.navbar, #nav-dropdown, .nav-item' ).addClass( 'solidcolor' );
			$( '.nav-link' ).addClass( 'white' );
			$( '#nav-dropdown .nav-link' ).removeClass( 'btn-outline-custom btn-pill' );
		} else {
			$( '.navbar, #nav-dropdown, .nav-item' ).removeClass( 'solidcolor' );
			$( '.nav-link' ).removeClass( 'white' );
			if ( $( window ).width() > 768 ) {
				$( '#nav-dropdown .nav-link' ).addClass( 'btn-outline-custom btn-pill' );
			}
		}
	} );

Operating system: Windows 10
Browser: Firefox, Chrome
Code editor: VS Code
Build system: Webpack 4.28
Bootstrap, Webpack + many other components, provided via NPM.

**Comment by mdo** [comment]
Only changes to navbars from v4.1.3 to v4.2.1 can be seen at 
https://github.com/twbs/bootstrap/compare/v4.2.1..v4.1.3#diff-df1f74faf207a474f1d78c86393221c8.

**Comment by foss-friendly** [comment]
> 
> 
> Only changes to navbars from v4.1.3 to v4.2.1 can be seen at
> https://github.com/twbs/bootstrap/compare/v4.2.1..v4.1.3#diff-df1f74faf207a474f1d78c86393221c8.

Thank you very much for Your reply! I've made a couple more experiments and now it's clear for me that this issue has nothing to do with Bootstrap, but with my own configuration and the way I've bundled my modules. I couldn't even recreate the problem in Code Pen, since the reduced test case was working properly. Once more, You have my gratitude for Your help!
